src/open_inwoner/react/components/MaterialIcons/MaterialIcons.stories.tsx

```diff
@@ -0,0 +1,236 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { MaterialIcon } from './MaterialIcons'
+import 'material-icons/iconfont/material-icons.css'
+
+const meta: Meta<typeof MaterialIcon> = {
+  title: 'React/MaterialIcons',
+  component: MaterialIcon,
+  parameters: {
+    layout: 'padded',
+    docs: {
```

Comment:
Instead of having all those separate stories-components for icons that do not really add much of value in a separate window, another option could be to create a bit of a drop down like so:

```
argTypes: {
  name: {
    control: { type: 'select' },
    options: ['home', 'search', 'settings', 'person', 'mail', 'description', 'inbox', 'inventory_2', 'group', 'help_outline', 'euro']
  }
}
```

Then you will need only 1 icon story and you can select different names. Maybe call it "Icon Showcase" .

Also maybe later we can add font-sizes and color variations.